app/controllers/user_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require './config/environment'
+
+#:nodoc:
+class UserController < Sinatra::Base
```

Comment:
Inherit ApplicationController  plox :)

Reply:
I can't get that to work without breaking the `ApplicationController`'s layout

Reply:
Meh Ship it!